services/QuillLMS/spec/requests/auth/learn_worlds_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Auth::LearnWorldsController do
+  subject { post auth_learn_worlds_courses_path }
+
+  let(:courses_endpoint) { stub_const('Auth::LearnWorlds::COURSES_ENDPOINT', 'https://learnworlds.com/courses') }
```

Comment:
Do you need this `stub_const` here (vs. just the url string). I'm not sure the constant is used in these tests.

Reply:
Good point.  Since I'm stubbing out the HTTParty call we don't need it.